tests/unit/random_sampling.c

```diff
@@ -175,6 +175,14 @@ void sample() {
     x = RNG_BINOM(5, 1);
     IGRAPH_ASSERT(x == 5);
 
+    x = RNG_BINOM((1LL << 31) - 1, 0.5);
+    IGRAPH_ASSERT(!igraph_is_nan(x));
+    IGRAPH_ASSERT(0 <= x && x <= (1LL << 31) - 1);
+
+    x = RNG_BINOM((1LL << 31), 0.5);
```

Comment:
`(igraph_integer_t) (1LL << 31)` will create a negative value when `igraph_integer_t` is 32-bit.

Reply:
Since this test is not printing, we could safely make it conditional on an `#if IGRAPH_INTEGER_SIZE > 32`

Reply:
oh, indeed, thanks.

Reply:
Fixed.